qa/integration-tests/src/test/java/io/camunda/it/operate/OperateMultiTenancyIT.java

```diff
@@ -38,10 +36,7 @@ public class OperateMultiTenancyIT {
 
   @ZeebeIntegration.TestZeebe
   private final TestStandaloneCamunda testInstance =
-      new TestStandaloneCamunda()
-          .withCamundaExporter()
-          .withAuthenticationMethod(AuthenticationMethod.BASIC)
-          .withMultiTenancyEnabled();
+      new TestStandaloneCamunda().withCamundaExporter().withBasicAuth().withMultiTenancyEnabled();
```

Comment:
I remember that there was a discussion to use the new extension to enable multi db testing ([CamundaMultiDBExtension](url)). In this case we have to use `TestSimpleCamundaApplication` instead of [TestStandaloneCamunda](url). Let's check if this Test can also be refactored to switch to the new extension or not.